singularity-eos/eos/eos_spiner_sie_transforms.hpp

```diff
@@ -0,0 +1,74 @@
+#ifndef _SINGULARITY_EOS_EOS_EOS_SPINER_SIE_TRANSFORMS_HPP_
+#define _SINGULARITY_EOS_EOS_EOS_SPINER_SIE_TRANSFORMS_HPP_
+
+#ifdef SINGULARITY_USE_SPINER_WITH_HDF5
+
+#include <type_traits>
+
+#include <algorithm>
+#include <cstdlib>
+#include <iomanip>
+#include <sstream>
+#include <string>
+#include <vector>
+// #include <iostream> // debug
+// #include <stdio.h> // debug
+
+// ports-of-call
+#include <ports-of-call/portability.hpp>
+
+// base
+#include <singularity-eos/eos/eos_spiner_common.hpp>
+
+namespace singularity {
```

Comment:
You might want to put these inside another namespace inside singularity like `transforms` or something similar.

Reply:
Seconded 🙂